Civi/Core/Container.php

```diff
@@ -192,7 +194,7 @@ public function createContainer() {
       // For Caches that we don't really care about the ttl for and/or maybe accessed
       // fairly often we use the fastArrayDecorator which improves reads and writes, these
       // caches should also not have concurrency risk.
-      $fastArrayCaches = ['groups', 'navigation', 'customData', 'fields', 'contactTypes', 'metadata', 'js_strings'];
+      $fastArrayCaches = ['groups', 'navigation', 'customData', 'fields', 'contactTypes', 'metadata', 'js_strings', 'mixinBoot', 'mixinScan'];
```

Comment:
This confused me a bit. Compare:

```php
/*A*/ CRM_Extension_System::getCache()->set('mixinBoot', 'yaddayadda');
/*B*/ Civi::cache('mixinBoot')->set('yadda', 'yadda');
```

To my reading, the `mixinBoot` is more like (A), and this patch would affect something like (B).